lib/iris/coords.py

```diff
@@ -909,14 +917,25 @@ def is_compatible(self, other, ignore=None):
         return compatible
 
     @property
-    def dtype(self):
+    def points_dtype(self):
```

Comment:
Don't we want to keep the `dtype` property, even if it returns the same as `points_dtype`, for backwards compatibility?